ncl/ncl_entries/climatology_functions.ipynb

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
<p>Currently throwing an error related to (?): </p>

<pre class="ql-syntax" spellcheck="false">&gt; mon_to_season = month_to_season(aice, "ASO")

Where <code>ValueError: The time axis length must be a multiple of 12.</code>
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Ah, shoot.  Somehow I missed this in swapping out the datafiles, but I know where it's coming from.  The NCL function (and Python one apparently) have some requirements for the time axis of the input dataset.

In other places I subset the data in the code to address this, but I think it's probably cleaner to go edit the datafile itself so we don't need to.

I'll go ahead and clean this up.

Reply:
This should be addressed by: https://github.com/NCAR/geocat-datafiles/pull/87